### PR TITLE
update from fedcloudlient

### PR DESCRIPTION
**Pull request by marcvs**
I wrote a script (https://github.com/EOSC-synergy/sla-monitor/blob/master/update-vo-operations.py) that uses
the output of `fedcloudclient.endpoint.get_projects_from_sites_dict` to update the content of this repository's sites/ folder.

This needs VO membership for the VO to be updated, so adding is safe, removing only possible for VOs where one is sure to be a member of. Therefore I'm currently only adding sites.

Since I expect synergy VOs to be a bit more dynamic, I've written this script.  We see three sites supporting a given VO only for two weeks of may.

### CODE REVIEW

**Review by enolfc — APPROVED** [review]

File: sites/NCG-INGRID-PT.yaml [hunk 1]
```diff
@@ -19,3 +19,15 @@ vos:
...
+  name: covid-synergy
+- auth:
+    project_id: a53ca78c534046e5b13f4537ae698411
+  name: worsica
+- auth:
+    project_id: ddf0c468c8af4e0bbb9808bfc0288381
+  name: eosc-synergy
+- auth:
+    project_id: 14e8f964db4c4a8ea286b0886ab51e7c
+  name: lago
```
Comment (enolfc) [inline sites/NCG-INGRID-PT.yaml:33]: There are some inconsistencies in the naming here

File: sites/NCG-INGRID-PT.yaml [hunk 2]
```diff
@@ -19,3 +19,15 @@ vos:
 - auth:
     project_id: a53ca78c534046e5b13f4537ae698411
   name: worsica.vo.incd.pt
+- auth:
+    project_id: 05e52356addc44e18ef2bd14f2e2f67d
+  name: covid-synergy
+- auth:
+    project_id: a53ca78c534046e5b13f4537ae698411
+  name: worsica
```
Comment (enolfc) [inline sites/NCG-INGRID-PT.yaml:27]: This was already above

**Review by enolfc — CHANGES_REQUESTED** [review]
Thanks @marcvs ! I was too fast approving before, there are some minor things to check:

- you shouldn't add VOs that are already there
- you should use the official VO name

File: sites/NCG-INGRID-PT.yaml (hunk 2 above)
Comment (marcvs) [reply sites/NCG-INGRID-PT.yaml:27]: Ok; please reject this PR, I'll create another one later today.

**Comment by marcvs** [comment]
The wrong VO names is a bug in the NCG-LIP config. I'll ask Mario to fix it.

File: sites/NCG-INGRID-PT.yaml (hunk 2 above)
Comment (marcvs) [reply sites/NCG-INGRID-PT.yaml]: Actually, this PR should work now.

File: sites/NCG-INGRID-PT.yaml (hunk 1 above)
Comment (marcvs) [reply sites/NCG-INGRID-PT.yaml]: Should be fixed

**Pull request closed without merging**